src/instruments/src/ND/elements/RadioNavInfo.tsx

```diff
@@ -11,11 +11,11 @@ export enum NavAidMode {
 
 export type RadioNavInfoProps = { index: 1 | 2, side: EfisSide }
 
-const TuningModeIndicator: React.FC<{ index: 1 | 2, frequency: number }> = ({ index, frequency }) => {
-    const [tuningMode] = useSimVar(`L:A32NX_FMGC_RADIONAV_${index}_TUNING_MODE`, 'enum');
+const TuningModeIndicator: React.FC<{ index: 1 | 2 }> = ({ index }) => {
+    const [tuningMode] = useSimVar('L:A32NX_FMGC_RADIONAV_TUNING_MODE', 'enum');
```

Comment:
This is incorrect. Each radio has a separate tuning mode.

Reply:
Hold on, there's many things wrong in this PR because I merged instead of rebased. Therefore I see changes from my other PR even though they are merged into the master. 

But yes indeed that's wrong. It's in stable/exp then